webapp/Connector/src/view/Learn.js

```diff
@@ -132,6 +132,124 @@ Ext.define('Connector.view.Learn', {
         }
     },
 
+    requestAssayExportCSV : function(cmp, item, data) {
+        console.log("export csv");
```

Comment:
was this just debug logging that can be removed?

Reply:
Done